superset-frontend/src/components/DvtSelect/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import Icon from '../Icons/Icon';
+import {
+  StyledSelect,
+  StyledSelectOption,
+  StyledSelectOptions,
+  StyledSelectLabel,
+  StyledSelectSelect,
+  StyledSelectIcon,
+} from './dvt-select.module';
+
+export interface DvtSelectProps {
+  label: string;
+  data: { value: string; label: string }[];
+  placeholder: string;
+  selectedValue: string;
+  setSelectedValue: (newSeletedValue: string) => void;
+}
+
+const DvtSelect: React.FC<DvtSelectProps> = ({
+  data,
+  label,
+  selectedValue,
+  setSelectedValue,
+}) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleSelectClick = () => {
+    setIsOpen(!isOpen);
+  };
+
+  const handleOptionClick = (value: string) => {
+    setSelectedValue(value);
+    setIsOpen(false);
+  };
+
+  return (
+    <StyledSelect onClick={handleSelectClick}>
+      <StyledSelectLabel>{label}</StyledSelectLabel>
+      <StyledSelectSelect isOpen={isOpen}>
+        {selectedValue}
+        <StyledSelectIcon>
+          {isOpen && (
+            <Icon fileName="caret_down" iconSize="xxl" iconColor="black" />
+          )}
+
+          {!isOpen && (
+            <Icon fileName="caret_right" iconSize="xxl" iconColor="black" />
```

Comment:
2 icon arrow right ve bottom beraber olmamalı. 1 tane icon olmalı ve icon gerisi div olan isOpen boolean olan animasyon right veya bottom conditional için olmalı